distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -1020,21 +1018,11 @@ type Builder
 
              Vector.new_builder
     new : Integer->Builder
-    new (capacity=1) = Builder (Array.new capacity) 0
-
-    ## Returns the current capacity (i.e. the size of the underlying storage)
-       of this builder.
-
-       > Example
-         Get the capacity of a new builder.
-
-             Vector.new_builder.capacity
-    capacity : Integer
-    capacity self = self.to_array.length
+    new (capacity=1) = Builder (ArrayList.new capacity)
```

Comment:
Btw. default `ArrayList` capacity is 10: https://github.com/openjdk/jdk/blob/343ecd806bb05065d31124573b3e58a864cfc617/src/java.base/share/classes/java/util/ArrayList.java#L118

Reply:
I assume you suggest that we should use the same default then?